version.php

```diff
@@ -25,7 +25,7 @@
 defined('MOODLE_INTERNAL') || die;
 
 $plugin->version = 2022031500;
-$plugin->requires  = 2020061500;
+$plugin->requires  = 2021052500;
```

Comment:
Do we have to change this for avoiding that this branch is being installed on a moodle <4.0?

Reply:
@PhMemmel  Yes, we need to change the requires for the 4.0 branch.  I am going to rename master to stable_39_311 and then rename the 4.0 branch to master.  Then I can close out this pull request.

Reply:
Can you just push this and do the branch-management or do you want me to do this? :-)